Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Compositor/ShaderManager.cs

```diff
@@ -0,0 +1,345 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#define OUTPUT_YUV
+
+using UnityEngine;
+using System.Collections;
+using System.Runtime.InteropServices;
+using System;
+using UnityEngine.Rendering;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView.Compositor
+{
+    public class ShaderManager : MonoBehaviour
```

Comment:
>ShaderManager [](start = 17, length = 13)

It feels like it may be more appropriate to call this texture manager

Reply:
Agreed, I'll change the name